### PR TITLE
#1813 Simple directions implementation

### LINKED ISSUE #1813: Add simple directions to a script item
## Is your feature request related to a problem? Please describe.

Cannot currently add any directions to items in a script.

## Describe the solution you'd like

Ability to add direections to items in a script!

## Implementation

N/A

## Describe alternatives you've considered

N/A

## Additional context

N/A

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #1813 

## Change summary

- Adds a simple directions implementation with a drop down for an items directions, and a textinput to type.

image

- Have created #1815 to tidy up `PrescriptionReducer.js`
- Sneaked in a fix in `incomingSyncUtils` to cast correctly.
- Added localization to issue
- Adds the correct field to display in the summary
- Adds a TextInput rather than a label for the displayed direction. Mainly this is for the simple implementation, as I dont see how this could work well when abbreviations are implemented. I think a modal would be needed which would have an additional drop down to show some directions to add, or something similar.
- The ScrollView wrap is to handle shifting the `TextInput` out of the way/above the keyboard

## Testing

N/A

### Related areas to think about

- Potentially better to just have `PrescriptionX` components all connected to redux. Actually pretty confident it would be! Not a huge shift

### CODE REVIEW

File: src/widgets/DropdownRow.js [hunk 1]
```diff
@@ -66,7 +63,13 @@ export const DropdownRow = ({
         />
       </View>
       <View style={localStyles.flexNine}>
-        <SimpleLabel size={labelSize} text={currentOptionText} />
+        <TextInput
+          onChangeText={onSelection}
+          value={currentOptionText}
+          underlineColorAndroid={SUSSOL_ORANGE}
+          style={{ flex: 1, fontFamily: APP_FONT_FAMILY }}
+          placeholder="Usage direction"
```
Comment (mark-prins) [inline src/widgets/DropdownRow.js]: this is very specific within a general component. A temporary thing?
I was expecting props to enable the text input and supply the placeholder

File: src/widgets/PopoverDropDown.js [hunk 2]
```diff
@@ -65,7 +67,7 @@ const DropDown = ({ anchorRef, isVisible, options, onSelection, onClose, title }
       onRequestClose={onClose}
       arrowStyle={arrowStyle}
       backgroundStyle={popoverBackgroundStyle}
-      placement="bottom"
+      placement="auto"
```
Comment (mark-prins) [inline src/widgets/PopoverDropDown.js]: perhaps the component should change its name from `PopoverDropDown`?? 😆

File: src/widgets/DropdownRow.js (hunk 1 above)
Comment (joshxg) [reply src/widgets/DropdownRow.js]: whoops! thanks : 1ac1107690ce2800fccf792bf8eb9e23906ea09c

File: src/widgets/PopoverDropDown.js (hunk 2 above)
Comment (joshxg) [reply src/widgets/PopoverDropDown.js]: 🤣  hmm.. `PopoverDropAuto` just doesn't seem to have the same ring to it

**Comment by joshxg** [comment]
Also removed some styles 
a74ed3f

**Pull request merged**